backend/editor/api.py

```diff
@@ -82,7 +95,7 @@ async def findAllNodes(response: Response):
     """
     Get all nodes within taxonomy
     """
-    result = get_all_nodes("")
+    result = get_all_nodes("", txn)

```

Comment:
Instead of passing txn around, if you put it in a graph_db.py file you can import it in entries.py and use it there. But beware as it is a global, you should not use:
```python
from .graph_db import current_transaction
...
current_transaction.do_something
```
Because you would miss updates.

You have to instead keep the namespace:

```python
from . import graph_db
...
graph_db.current_transaction.do_something
```